ynr/apps/uk_results/migrations/0054_migrate_is_winner_to_elected.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.18 on 2021-05-13 15:33
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    """
+    Ensures that for every candidate result we have we store the definitive True
+    or False value for if they were elected against the related candidacy before
+    we throw this data away in next migration to remove the field
+    """
+    CandidateResult = apps.get_model("uk_results", "CandidateResult")
```

Comment:
I have refactored this in a later commit to avoid using a loop and avoid the modified timestamps being updated